DuckDuckGo/MainViewController.swift

```diff
@@ -64,6 +64,25 @@ class MainViewController: UIViewController {
         }
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        showiOS11Popup()
+    }
+    
+    private func showiOS11Popup() {
+        var settings = TutorialSettings()
+        if !settings.hasSeenOnboarding, settings.hasSeeniOS11Popup {
```

Comment:
The popup shows every time the app is restarted.  

I believe this syntax is equivalent to `&&` but you probably want this to be a `||`?